storage/BasicColumnStoreTupleStorageSubBlock.cpp

```diff
@@ -173,45 +173,45 @@ BasicColumnStoreTupleStorageSubBlock::BasicColumnStoreTupleStorageSubBlock(
   }
 }
 
-bool BasicColumnStoreTupleStorageSubBlock::DescriptionIsValid(
+int BasicColumnStoreTupleStorageSubBlock::DescriptionIsValid(
     const CatalogRelationSchema &relation,
     const TupleStorageSubBlockDescription &description) {
   // Make sure description is initialized and specifies BasicColumnStore.
   if (!description.IsInitialized()) {
-    return false;
+    return -1;
```

Comment:
IMHO, we don't need to change the return value. Instead, for the debug purpose, we could print out the proto content:

```C++
DCHECK(BasicColumnStoreTupleStorageSubBlock::ProtoIsValid(relation_, description_))
    << "Attempted to create BasicColumnStoreTupleStorageSubBlock from an invalid proto description:\n"
    << proto.DebugString();
```